### DISCUSSION TITLE
Is it possible load a zip file into memory and use ZipArchive to read it while compressed?

### DISCUSSION THREAD

**Discussion by Sahasrara** [dotnet/runtime#95558] [Q&A] [answered]
I'd like to keep a compressed zip file in memory and use ZipArchive to read it as though it were on disk. Is that possible?

**Comment by elgonzo** [comment, marked as answer]
Yes, of course. A `System.IO.Compression.ZipArchive` instance operates on a `System.IO.Stream` instance, and that includes instances of `System.IO.MemoryStream`.
